Dockerfile

```diff
@@ -1,20 +1,41 @@
-FROM phusion/baseimage:latest
+FROM edxops/xenial-common:latest
+MAINTAINER stv <stv@stanford.edu>
 RUN apt-get update && apt-get install -y \
+    apt-transport-https \
     build-essential \
     git \
     lib32z1-dev \
     libjpeg62-dev \
     libxml2-dev \
     libxslt-dev \
+    libz-dev \
     python-dev \
     python-setuptools \
+    python-distribute \
+    python-pip \
+    python3-dev \
+    python3-pip \
     xz-utils \
 && rm -rf /var/lib/apt/lists/*
+
+ADD bin/nodejs-setup /tmp/nodejs-setup
```

Comment:
Some scripts do this by `curl`ing a file down and executing it;
I'd rather store the file locally.